lib/vanity/configuration.rb

```diff
@@ -48,6 +48,12 @@ def default_request_filter(request) # :nodoc:
       request_filter: ->(request) { default_request_filter(request) },
       templates_path: File.expand_path(File.join(File.dirname(__FILE__), 'templates')),
       use_js: false,
+      cookie_name: 'vanity_id',
+      cookie_expires: 20.years,
```

Comment:
The `.years` is a Rails only extension, mind expanding this to something like `20 * 365 * 24 * 60 * 60`?